agagd/templates/agagd_core/game_count_graph.html

```diff
@@ -79,11 +79,11 @@
       .call(yAxis2)
     .append("text")
       .attr("transform", " rotate(-90)")
-      .attr("x", -height)
+      .attr("x", -85)
       .attr("y", -16)
       .attr("dy", ".71em")
-      .style("text-anchor", "end")
-      .text("total # of Games");
+      .style("text-anchor", "start")
+      .text("Total # of Games");
```

Comment:
Looking at this a second time. I don't think we need the "Total # of Games" or "# of Games" on both sides. I think we could do with just the one on the left with your new label "Total # of Games" vs "# of Games."

Reply:
One has been hidden so people dont like the graph. Trying to fix the graph to correctly show what it is until we can overhaul it with other info as per #122 .

Reply:
My issue is that we are introducing a new issue here, so now we have the label on both sides. One of which may be obscured by our graph in production.
![image](https://user-images.githubusercontent.com/5054653/103593300-54db5800-4eaa-11eb-85a4-b9e89c84cea0.png)

Reply:
I think I was playing with it on the test server and then replicating it for my git files tbh. The right side test should still sit below the graph lines.

Reply:
I checked the test server, and I am seeing an error 500, so I have reverted that. If you would like to mess with this locally, try doing so with `$ docker-compose up --build --force recreate`. You should be able to download the schema from https://github.com/usgo/usgo-sql-schemas. There are instructions for basic docker-compose operations on: https://github.com/usgo/agagd/blob/master/docs/docker.md. The only two files that are really needed to run `$ docker-compose up` are: schema.sql and config-docker.env. It may be finicky on the values for config-docker.env. Let me know if you have any issues. 

Otherwise, the change does make sense.